cmd/aws-cloud-controller-manager/main.go

```diff
@@ -40,25 +40,25 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/wait"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/cloud-provider/app"
+	"k8s.io/cloud-provider/options"
 	"k8s.io/component-base/cli/globalflag"
 	"k8s.io/component-base/logs"
 	_ "k8s.io/component-base/metrics/prometheus/clientgo" // for client metric registration
 	_ "k8s.io/component-base/metrics/prometheus/version"  // for version metric registration
 	"k8s.io/klog"
-	"k8s.io/kubernetes/cmd/cloud-controller-manager/app"
-	"k8s.io/kubernetes/cmd/cloud-controller-manager/app/options"
 	"k8s.io/kubernetes/pkg/features" // add the kubernetes feature gates
```

Comment:
@cici37 what's the recommended home for the feature gates now?

Reply:
The cloud related feature gates: `"k8s.io/controller-manager/pkg/features/register"`

Reply:
`"k8s.io/controller-manager/pkg/features/register"` does not contain `IPv6DualStack` feature that we used here, will we add `IPv6DualStack` in it in the future? Or we keep using `"k8s.io/kubernetes/pkg/features"`?

Reply:
`IPv6DualStack` is used as an local copy in CCM now : `https://github.com/kubernetes/kubernetes/blob/d9b7e64b6115dbfc83b844a13a4d1d7a8861a293/staging/src/k8s.io/cloud-provider/app/core.go#L44`

Reply:
The goal should be to use k8s.io/controller-manager/pkg/features/register. If we find anything missing we would move it to an appropriate public place. Fundamentally we should have gotten rid of all "k8s.io/kubernetes/*" imports. In this case I believe Cici has the answer.